bors.toml

```diff
@@ -1,5 +1,5 @@
 status = [
-  "azure-ci",
+  "test-os",
```

Comment:
```suggestion
  "test-os (ubuntu-latest)",
  "test-os (windows-latest)",
  "test-os (macos-latest)",
```

Reply:
I think it reads the names list on the left side of https://github.com/CliMA/ClimateMachine.jl/runs/766357298